textures.cfg

```diff
@@ -140,12 +140,18 @@ setshaderparam "specscale" 0.1
     texture z "dziq/eco_floor_z"
 
 // glass_alpha01
-setshader bumpspecalphaworld
+setshader bumpspecmapalphaworld
     texture c "dziq/glass_alpha01" 0 0 0 0.25
     texture n "dziq/glass_alpha01_n"
     texture s "dziq/glass_alpha01_s"
     texture a "dziq/glass_alpha01_a"
 
+// glass_alpha02
+setshader bumpspecalphaworld
```

Comment:
Is this supposed to be `bumpspecmapalphaworld`?

Reply:
Hmm it's supposed to be without specmap,   but might look not so proper so I added a specmap